### PR TITLE
update charts to v0.3.0

### PR DESCRIPTION
**Pull request by moshe010**
update network operator app to v0.3.0
update network operator chart to v0.3.0
update nfd chart to v0.2.0
update Readme nfd to v0.6.0-233-g3e00bfb

Signed-off-by: Moshe Levi <moshele@nvidia.com>

### CODE REVIEW

**Review by e0ne — COMMENTED** [review]
Why do we need to bump NFD chart to 0.2.0?

**Review by e0ne — APPROVED** [review]

File: deployment/network-operator/Chart.yaml [hunk 1]
```diff
@@ -15,12 +15,9 @@ dependencies:
 - condition: nfd.enabled
   name: node-feature-discovery
   repository: ""
-  version: 0.1.0
+  version: 0.2.0
```
Comment (adrianchiris) [inline deployment/network-operator/Chart.yaml:18]: we should keep the same chart version as in NFD repo IMO

**Review by adrianchiris — COMMENTED** [review]
multus CNI default version under `chart parameters` should be updated to v3.6.
in _deployment/network-operator/README.md_

**Comment by adrianchiris** [comment]
#117

**Pull request merged**